tests/__fixtures__/data/filtersearch.ts

```diff
@@ -1,43 +1,59 @@
-export const unsectionedFilterSearchResponse = {
+import { FilterSearchResponse, Matcher } from '@yext/answers-headless-react';
+
+export const unsectionedFilterSearchResponse: FilterSearchResponse = {
```

Comment:
I see-- I think one of the important difference between the data is that one has labeled sections and one has unlabeled sections.  I can for sure rename the objs, but I'm worried that it will mess up for FilterSearch stories since it also uses  this data.

Reply:
@alextaing renaming won't mess up the stories since actual object will stay the same.

Reply:
I was also going to comment on the boolean value of sectioned-- could we discuss it a little during our check-in?